src/app/helptext/jails/jails-add.ts

```diff
@@ -14,6 +14,9 @@ specified RELEASE directories as nullfs mounts over the jail \
 directories. Basejails are not linked to the original RELEASE \
 when upgraded.'),
 
+https_placeholder: T('Fetch release via HTTPS'),
+https_tooltip: T(''),
```

Comment:
via should be used, it's operating over the protocol HTTP *via* SSL.

Reply:
No, via is a road or route.  It's not clear to use it here, because it implies movement.  HTTPS is not moving a download with SSL, it is encrypting it.